extension/chrome/settings/setup.ts

```diff
@@ -123,6 +123,11 @@ export class SetupView extends View {
       opgp.config.showComment = false;
       opgp.config.showVersion = false;
     }
+    if (this.clientConfiguration.shouldAllowInsecureSha1Hash() && typeof opgp !== 'undefined') {
+      opgp.config.rejectHashAlgorithms = new Set([...defaultRejectedHashAlgo]);
+    } else {
+      opgp.config.rejectHashAlgorithms = new Set([...defaultRejectedHashAlgo, opgp.enums.hash.sha1]);
+    }
```

Comment:
Hi @martgil, I think we shouldn't completely disable SHA1 algorithm with `rejectHashAlgorithms` property, as we already show error `error verifying signature: Insecure hash algorithm: SHA1. Sender is using old, insecure OpenPGP software.` when user verifies message signed with sha1 key.
Disabling sha1 entirely would also make it impossible for users to decrypt older messages encrypted with sha1 keys.
So let's just remove `opgp.config.rejectHashAlgorithms` property altogether.

Reply:
I understand completely, Roma, sorry. I'll be closing this PR and proceed with removing the `rejectHashAlgorithms` property in openpgp configuration.